examples/link_prediction/ogbl-ppa/full_graph_link_predictor.py

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.data import DataLoader
+
+from ogb.linkproppred import DglLinkPropPredDataset, Evaluator
+from logger import Logger
+
+from dgl.nn.pytorch import GraphConv
+from dgl.nn.pytorch.conv import SAGEConv
+
+class GCN(nn.Module):
```

Comment:
No need to define a GCN class here and you should use the one from dgllife directly with

```python
from dgllife.model import GCN
```